lib/async/als.js

```diff
@@ -29,10 +30,16 @@ export const getStore = () => MontiAsyncStorage.getStore() ?? {};
 
 export const getInfo = () => MontiAsyncStorage.getStore()?.info;
 
-export const setActiveEvent = (event, store = MontiAsyncStorage.getStore()) => {
+export const mergeEnterWith = (partial = {}) => {
+  const store = MontiAsyncStorage.getStore();
+
   if (!store) return;
 
-  MontiAsyncStorage.enterWith({ ...store, activeEvent: event });
+  MontiAsyncStorage.enterWith(pipe({}, merge(store), merge(partial)));
```

Comment:
Note to self, check performance vs. es6 syntax.

Reply:
It might be simpler with es5:

```suggestion
  MontiAsyncStorage.enterWith(Object.assign({}, store, partial));
```

Reply:
Yep, mind glitch, really wanted to use Remeda somewhere I guess.